### ISSUE TITLE
Project Section is not accessible using the keyboard: A11y_.NET 9 template_.NET 9 template_Project_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26124]
### GitHub Tags
#A11yTCS; #A11ySev1; #Win32; #WCAG2.1.1; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #Keyboard; #A11yMAS;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till projects section.
4. Observe that Project Section is not accessible using the keyboard: 

### Similar issue is also observed for tags.

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till project section, and press enter on any project, Project screen will open, 
4. Tab till tags.
5. Observe that tags are not accessible using the keyboard.

### Similar issue is also observed for Projects options.

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter, Projects options will screen will open.
4. Observe that Projects options are not accessible using the keyboard.

### Actual Result:
Project Section is not accessible using the keyboard.

### Expected Result:
Project Section should be accessible using the keyboard: 

### User Impact:
keyboard users will not be able to access the controls.

### Attachment:

https://github.com/user-attachments/assets/9e36bc43-11a0-4248-b010-7afc14bfd689
Similar issue1
https://github.com/user-attachments/assets/c7e37233-061b-41b1-a663-d55bea964864

Similar issue2
https://github.com/user-attachments/assets/ec38b399-cb93-44fd-b2e0-8e843fc9c738
